### PR TITLE
web: add link to source code in About Section in Setting

### LINKED ISSUE #3657: Add link to Source code (Github) to About section in Settings.
I would like to have link to source code in Settings, not to issues, not to something else, but to source code. Thank you :)

### PR DESCRIPTION
**Pull request by inumaki**
By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Aryan aryankatiyar1420@gmail.com

fixes #3657

### CODE REVIEW

**Comment by inson1** [comment]
@alihamuh @alihamuh @thecodrr What do you think?

**Comment by inson1** [comment]
@inumaki Btw you dont have to write so many thinks + I'm not admin or anything, just normal user who had idea

File: apps/web/src/dialogs/settings/other-settings.ts [hunk 1]
```diff
@@ -72,6 +72,23 @@ export const AboutSettings: SettingsGroup[] = [
           ];
         }
       },
+      {
+        key: "source-code",
+        title: "Source Code",
```
Comment (thecodrr) [inline apps/web/src/dialogs/settings/other-settings.ts:77]:
```suggestion
        title: "Source code",
```

File: apps/web/src/dialogs/settings/other-settings.ts [hunk 2]
```diff
@@ -72,6 +72,23 @@ export const AboutSettings: SettingsGroup[] = [
...
+        description: "Check out our source code",
```
Comment (thecodrr) [inline apps/web/src/dialogs/settings/other-settings.ts:78]:
```suggestion
        description: "All the source code for Notesnook is available & open for everyone on GitHub.",
```

File: apps/web/src/dialogs/settings/other-settings.ts [hunk 3]
```diff
@@ -72,6 +72,23 @@ export const AboutSettings: SettingsGroup[] = [
...
+        components: [
+          {
+            type: "button",
+            action: () =>
+              void window.open(
+                "https://github.com/streetwriters/notesnook",
+                "_blank"
+              ),
+            title: "Check Source Code",
```
Comment (thecodrr) [inline apps/web/src/dialogs/settings/other-settings.ts:87]:
```suggestion
            title: "Check source code",
```

File: apps/web/src/dialogs/settings/other-settings.ts [hunk 4]
```diff
@@ -72,6 +72,23 @@ export const AboutSettings: SettingsGroup[] = [
...
       },
+      {
+        key: "source-code",
+        title: "Source code",
+        description: "All the source code for Notesnook is available & open for everyone on GitHub.",
+        components: [
+          {
+            type: "button",
+            action: () =>
+              void window.open(
```
Comment (thecodrr) [inline apps/web/src/dialogs/settings/other-settings.ts:83]:
```suggestion
              window.open(
```

File: apps/web/src/dialogs/settings/other-settings.ts [hunk 5]
```diff
@@ -72,6 +72,23 @@ export const AboutSettings: SettingsGroup[] = [
...
+                "https://github.com/streetwriters/notesnook",
+                "_blank"
+              ),
```
Comment (thecodrr) [inline apps/web/src/dialogs/settings/other-settings.ts:86]:
```suggestion
            action: () => {
                window.open(
                  "https://github.com/streetwriters/notesnook",
                  "_blank"
                );
              },
```

**Pull request merged**